### PR TITLE
chore: update bulk-api-adapter and mojaloop-simulator

### PR DESCRIPTION
**Pull request by kleyow**
```
Testing Toolkit Report
Test Result: PASSED
Test Suite: GP Tests
Environment: moja2.test.mojaloop.live
Total assertions: 2573
Passed assertions: 2573
Failed assertions: 0
Total requests: 587
Total test cases: 129
Passed percentage: 100.00%
Started time: Fri, 08 Jul 2022 17:12:01 GMT
Completed time: Fri, 08 Jul 2022 17:15:05 GMT
Runtime duration: 183461 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-08T17%3A15%3A05.035Z.html)

```
Testing Toolkit Report
Test Result: PASSED
Test Suite: Bulk Tests
Environment: moja2.test.mojaloop.live
Total assertions: 183
Passed assertions: 183
Failed assertions: 0
Total requests: 17
Total test cases: 4
Passed percentage: 100.00%
Started time: Fri, 08 Jul 2022 17:15:21 GMT
Completed time: Fri, 08 Jul 2022 17:15:45 GMT
Runtime duration: 24115 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-08T17%3A15%3A45.392Z.html)

```
Testing Toolkit Report
Test Result: PASSED
Test Suite: Thirdparty Tests
Environment: moja2.test.mojaloop.live
Total assertions: 21
Passed assertions: 21
Failed assertions: 0
Total requests: 20
Total test cases: 8
Passed percentage: 100.00%
Started time: Fri, 08 Jul 2022 17:16:22 GMT
Completed time: Fri, 08 Jul 2022 17:16:27 GMT
Runtime duration: 4727 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-08T17%3A16%3A27.007Z.html)

### CODE REVIEW

**Review by elnyry-sam-k — APPROVED** [review]

**Comment by kleyow** [comment]
```
Testing Toolkit Report
Test Result: PASSED
Test Suite: GP Tests
Environment: moja2.test.mojaloop.live
Total assertions: 2573
Passed assertions: 2573
Failed assertions: 0
Total requests: 587
Total test cases: 129
Passed percentage: 100.00%
Started time: Mon, 11 Jul 2022 15:57:03 GMT
Completed time: Mon, 11 Jul 2022 15:59:53 GMT
Runtime duration: 170795 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-11T15%3A59%3A53.892Z.html)

```
Testing Toolkit Report
Test Result: PASSED
Test Suite: Bulk Tests
Environment: moja2.test.mojaloop.live
Total assertions: 183
Passed assertions: 183
Failed assertions: 0
Total requests: 17
Total test cases: 4
Passed percentage: 100.00%
Started time: Mon, 11 Jul 2022 16:00:18 GMT
Completed time: Mon, 11 Jul 2022 16:00:42 GMT
Runtime duration: 23868 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-11T16%3A00%3A42.064Z.html)

```
Testing Toolkit Report
Test Result: PASSED
Test Suite: Thirdparty Tests
Environment: moja2.test.mojaloop.live
Total assertions: 21
Passed assertions: 21
Failed assertions: 0
Total requests: 20
Total test cases: 8
Passed percentage: 100.00%
Started time: Mon, 11 Jul 2022 16:01:27 GMT
Completed time: Mon, 11 Jul 2022 16:01:31 GMT
Runtime duration: 4095 ms
```
[View Report](https://mojaloop-oss-qa-results.s3.us-west-2.amazonaws.com/ttk-tests/reports/TTK-Assertion-Report-multi-2022-07-11T16%3A01%3A31.945Z.html)

File: mojaloop-simulator/values.yaml [hunk 1]
```diff
@@ -503,7 +503,7 @@ defaults: &defaults
         # state you can use a preexisting file. If running in a container, you can mount a sqlite file as a
         # volume in the container to preserve state between runs.
         # Use MODEL_DATABASE: :memory: for an ephemeral in-memory database
-        MODEL_DATABASE: ./model.sqlite
+        MODEL_DATABASE: ':memory:'
```
Comment (kleyow) [inline mojaloop-simulator/values.yaml:506]: `./model.sqlite` seems to be somehow generated when running the application but during the upgrade this no longer seems to be generated. I can't find anything in the sqlite packages noting a change. It should be fine to use memory here I believe.

File: mojaloop-simulator/values.yaml [hunk 2]
```diff
@@ -490,7 +490,7 @@ defaults: &defaults
         # The name of the sqlite log file. This probably doesn't matter much to the user, except that
         # setting :memory: will use an in-memory sqlite db, which will be faster and not consume disk
         # space. However, it will also mean that the logs will be lost once the container is stopped.
-        SQLITE_LOG_FILE: ./log.sqlite
+        SQLITE_LOG_FILE: ':memory:'
```
Comment (kleyow) [inline mojaloop-simulator/values.yaml:493]: `./log.sqlite` - before the upgrade, sqlite seemed fine to accept a file that doesn't exist, but now will throw a error if file is missing. It should be fine to use memory here as well.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**